plugins/poll/config/locales/server.en.yml

```diff
@@ -58,6 +59,7 @@ en:
     min_vote_per_user:
       one: A minimum of %{count} vote is required for this poll.
       other: A minimum of %{count} votes is required for this poll.
+    irv_vote_options_mismatch: "For IRV, the number of options supplied in a vote must match the number of options.  The number of options is %{count} and the number provided was %{provided}."
```

Comment:
Please avoid using acronyms like IRV. This makes it harder to understand for users and translators.

Reply:
this is done